encord/metadata_schema.py

```diff
@@ -41,10 +42,23 @@ class _ClientMetadataSchemaTypeEnum(BaseModel):
     values: Sequence[str] = Field([], min_length=1, max_length=256)
 
 
+class _ClientMetadataSchemaTypeEmbeddingApi(BaseModel):
+    # FIXME: remove unsupported field layouts once finalised.
+    url: Optional[str] = None
```

Comment:
should we only keep the ones we're sure about here to avoid breaking changes in the future? (only url)